### PR TITLE
feat: disable document driven with route meta

### PR DESCRIPTION
**Pull request by farnabaz**
### ❓ Type of change

- [ ] 📖 Documentation (updates to the documentation or readme)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [x] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description
Allow disabling/modifying document-driven 
```vue
<script setup>
definePageMeta({
  documentDriven: false,
  // Or if need content but no surround
  documentDriven: {
    page: true,
    surround: false,
  },
});
</script>
```

### 📝 Checklist

- [ ] I have linked an issue or discussion.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/runtime/plugins/documentDriven.ts [hunk 1]
```diff
@@ -46,6 +46,12 @@ export default defineNuxtPlugin((nuxt) => {
   }
 
   const refresh = async (to: RouteLocationNormalized | RouteLocationNormalizedLoaded, force: boolean = false) => {
+    const routeConfig = (to.meta.documentDriven || {}) as any
+    // Document drive disabled on route
```
Comment (Atinux) [inline src/runtime/plugins/documentDriven.ts:50]:
```suggestion
    // Disabled document driven mode on next route
```

**Comment by Atinux** [comment]
Looking great thank you!

Can you update the documentation on https://content.nuxtjs.org/guide/writing/document-driven to add a "Disable or control the page data" section before the "Example" section?

File: docs/content/3.guide/1.writing/7.document-driven.md [hunk 2]
```diff
@@ -160,6 +160,38 @@ const { theme } = useContent().globals
...
+<script setup lang="ts">
+definePageMeta({
+  documentDriven: false
+})
+</script>
+```
+
+### Control data
+
+To control docuemnt driven data you can pass an object to `documentDriven` meta key and enable/disable specific parts of it.
```
Comment (hacknug) [inline docs/content/3.guide/1.writing/7.document-driven.md:181]: Small type. Says `docuemnt` instead of `document` 👍

**Review by hacknug — COMMENTED** [review]
Just a small type

**Review by Atinux — APPROVED** [review]

**Pull request merged**